internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -473,6 +473,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 
 	cm.SetConditions(xpv1.ReconcileSuccess())
 
+	// Copy any custom status conditions from the XR to the claim.
+	for _, cType := range xr.GetClaimConditions() {
+		c := xr.GetCondition(cType)
+		cm.SetConditions(c)
+	}
+
```

Comment:
I don't think it is necessary. I just added `Healthy` to the runtime changes because it seemed to have no negative impact if added as a "system condition" prior to the actual implementation of `Healthy`. And this prevents us from needing to remember to make the change in the future.

Reply:
Got it. In that case I suggest adding a TODO with a link to the tracking issue noting that it's not implemented yet, or perhaps just removing it for now. There's a risk it takes us a long time to get around to implementing it (or less likely, we change our mind) in which case it would be confusing.

Reply:
I added the TODO to the runtime variable. I am leaning toward keeping it as this will effectively reserve `Healthy` for system use. If not, then someone may implement `Healthy` as one of their custom conditions, and would encounter trouble when the system `Healthy` condition rolls out in the future.